src/commands/LettuceMeetVisualizer.ts

```diff
@@ -0,0 +1,292 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { CommandInteraction, EmbedFieldData, MessageEmbed } from 'discord.js';
+import Command from '../Command';
+import { BotClient } from '../types';
+import axios from 'axios';
+
+// URL for Lettuce Meet GraphQL API, and the query to get the event data.
+const URL = 'https://api.lettucemeet.com/graphql';
+const query = `query EventQuery($id: ID!) {
+  event(id: $id) {
+      ...Event_event...EditEvent_event id
+  }
+}
+fragment EditEvent_event on Event {
+  id title description type pollStartTime pollEndTime 
+  maxScheduledDurationMins pollDates isScheduled start end timeZone updatedAt
+}
+fragment Event_event on Event {
+  id title description type pollStartTime pollEndTime maxScheduledDurationMins 
+  timeZone pollDates start end isScheduled createdAt updatedAt user {
+      id
+  }
+  googleEvents {
+      title start end
+  }
+  pollResponses {
+      id user {
+          __typename...on AnonymousUser {
+              name email
+          }...on User {
+              id name email
+          }...on Node {
+              __isNode: __typename id
+          }
+      }
+      availabilities {
+          start end
+      }
+      event {
```

Comment:
Do we need all the items in this query? If not, we can leave some of them out. Also we should try to keep indentation since it makes it easier to read